csrc/python_frontend/fusion_definition.cpp

```diff
@@ -124,11 +124,25 @@ void FusionDefinition::setupSchedule(const at::ArrayRef<c10::IValue>& inputs) {
   // original and not the copy needed for scheduling.
   buildFusionIr(user_sched_->schedule.get());
 
+  KernelArgumentHolder args =
+      KernelArgumentHolder::createKernelArgumentHolder(inputs, device);
+
+  // Concretize fusion
+  DynamicTransformInitialInfo initial_info =
+      DynamicTransform::getInitialInfo(user_sched_->schedule.get());
+  ExpressionEvaluator expr_eval =
+      executor_utils::bindInputs(args, user_sched_->schedule.get());
+  std::unique_ptr<DynamicTransformConcretizationInfo> conc_info =
+      std::make_unique<DynamicTransformConcretizationInfo>(
+          &initial_info, &expr_eval);
+  DynamicTransform::concretizeFusion(
+      user_sched_->schedule.get(), conc_info.get());
+
```

Comment:
As of #2465 this pattern is enshrined:
```suggestion
  // Concretize fusion
  DynamicTransform::concretizeFusion(user_sched_->schedule.get(), inputs);

```
Since you are re-using `args` below, we could add another version that takes a `KernelArgumentHolder` instead of a `const std::vector<c10::IValue>&`.